.vscode/settings.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
@ahl I believe it's correct to remove this file now.  I think we did the same for omicron when we switch to stable rustfmt.